scripts/accounts/gcloud/verify-principal-roles-and-permissions.sh

```diff
@@ -9,7 +9,7 @@ helpFunction()
     echo ""
     echo "Arguments:"
     echo "  -g     [Required] Google Account of an end user. Mutually exclusive with -s."
-    echo "  -s     [Required] Service Account name. Mutually exclusive with -g."
+    echo "  -s     [Required] Service Account email. Mutually exclusive with -g."
```

Comment:
Please accept both:

````
[[ "$serviceAccount" =~ .*"@".* ]] && service_account_email="$service_account@$project_id.iam.gserviceaccount.com" || service_account_email="$serviceAccount"
````

Reply:
I think you meant this:
```suggestion
    [[ "$serviceAccount" =~ .*"@".* ]] && service_account_email="$serviceAccount" || service_account_email="$service_account@$project_id.iam.gserviceaccount.com"  
```

Reply:
I forgot about this script, now it is done.